tritonsrc/attn_torch_function.py

```diff
@@ -51,49 +52,43 @@ def tuned_attn_fwd(
     stride_kz, stride_kh, stride_kn, stride_kk,
     stride_vz, stride_vh, stride_vk, stride_vn,
     stride_oz, stride_oh, stride_om, stride_on,
-    stride_bz, stride_bh, stride_bm, stride_bn,
-    cu_seqlens_q, cu_seqlens_k,
-    max_seqlens_q, max_seqlens_k,
-    head_dim_q, head_dim_k,
+    seqlen_q,
+    seqlen_k,
+    head_dim,
     dropout_p,
     philox_seed,
     philox_offset_base,
     encoded_softmax,
-    VARLEN: tl.constexpr,
```

Comment:
I take it this will be brought back or addressed on the next kernel merge?

Reply:
Yes. For now it's no point to keep parameters that does not exist in the Triton kernel.